### ISSUE TITLE
Add historic districts

### ISSUE THREAD

**Issue by SashaWeinstein** [NYCPlanning/db-data-library#199]
Used as source data for equitable development data tool. Downloaded from https://data.cityofnewyork.us/Housing-Development/Historic-Districts/xbvj-gfnw

**Comment by SashaWeinstein** [comment]
I'm basing my work off the `doitt_buildingfootprints_historical` dataset. It also pulls from a socrata dataset where multi-polygons are stored in a `the_geom` column. @AmandaDoyle if there's some reason not to base my work off this dataset let me know

**Comment by SashaWeinstein** [comment]
**lpc_historic_districts dataset already exists**
Ran into the issue yesterday that there is already lpc_historic_districts dataset. This pulls from the [LPC Individual Landmark and Historic District Building Database](https://data.cityofnewyork.us/Housing-Development/LPC-Individual-Landmark-and-Historic-District-Buil/7mgd-s57w). This dataset has more granular building-by-building information for buildings within historic districts and also information on individual landmarks. 

My mistake yesterday was creating a new lpc_historic_districts.yml that overwrote the existing .yml. I then overwrote the existing information on *historic buildings* in digital ocean with the new dataset on *historic districts*. I then didn't realize that data library has a system of sending cached data from `/.library` to digital ocean so I just kept sending the cached data on historic districts over and over again until I realized I needed to delete the cached information. 

All this took me around an hour to figure out and fix and I made no progress on creating the new dataset. Documenting this here because I want to remember that this happened and also I there are two issues with data library I want to flag
1. The existing lpc_historic_districts dataset is misnamed. It should be called lpc_historic_buildings. I don't think it's worth the effort the change it in data library, digital ocean, and everywhere its imported though.
2. It's hard to see that data library uses a caching system and hard to figure out how it works. One improvement I was thinking of here was adding a print statement that tells the user when they are reading from the `/.library` cache. Figuring out where to put this print statement would also hep me figure out the design
